City-of-Helsinki/helsinki-paatokset
# Pp 134 policymaker members block

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-134

**How to test**

Checkout branch and run `make shell` -> `drush cim -y; drush cr`
Make sure you have some policymakers and the latest meeting data (if not run` drush ap:fs -v` and `drush mim ahjo_meetings:latest; drush mim ahjo_decisionmakers:all`)

Go to i.ex. https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto and scroll to the bottom of the page. There should now be the members block visible. Keep in mind that the data shown at the moment is dummy data that will be replaced with actual data later. 

Test the functionality of The Vue application: try searching, sorting etc. Check that the styles match in desktop and mobile versions.

Check the code that everything looks good. Could something in the js file be written better?
